src/main/java/com/trailfinder/service/IHikeService.java

```diff
@@ -16,31 +16,31 @@
 public interface IHikeService {
 
 	/*
```

Comment:
Removed 'shoulds' from comments as the functionality of the methods are implied.

Reply:
Agreed.